labcodes/labstorybook
# DSYS-26 - Cards: first pass on React components

## Link to task:
https://labcodes.atlassian.net/browse/DSYS-26

## How to test:

With the storybook running in localhost, go to the Component/Card page, then check if everything is rendered as expected. No visual changes will be triggered, so you'll need to open the dev tools to see the css classes changing.
